packages/components/psammead-brand/src/index.stories.jsx

```diff
@@ -18,10 +18,19 @@ storiesOf('Brand', module)
         .map(key => key.charAt(0).toUpperCase() + key.slice(1));
 
       const choice = select('Service SVG', options, 'news').toLowerCase();
-      return <Brand brandName="Default Brand Name" svg={svgs[choice]} />;
+      return (
+        <Brand
```

Comment:
Can we name this story something a bit more descriptive? `with brand link` should suffice. 

This story doesn't reflect the default behaviour of the component, as the default behaviour is when `url` is null, which then renders a different component.